content/en/docs/concepts/signals/baggage.md

```diff
@@ -52,23 +55,37 @@ then this risk may not apply, but keep in mind that downstream services could
 propagate Baggage outside your network.
 
 Also, there are no built-in integrity checks to ensure that Baggage items are
-yours, so exercise caution when retrieving them.
+yours, so exercise caution when reading them.
 
-## Baggage is not the same as Span attributes
+## Baggage is not the same as attributes
 
-One important thing to note about Baggage is that it is not a subset of the
-[Span Attributes](/docs/concepts/signals/traces/#attributes). When you add
-something as Baggage, it does not automatically end up on the Attributes of the
-child system’s spans. You must explicitly take something out of Baggage and
-append it as Attributes.
+An important thing to note about baggage is that it is a separate key-value
+store and is unassociated with attributes on spans, metrics, or logs.
 
-For example, in .NET you might do this:
+To add Baggage entries to attributes, you need to explicitly read that data and
+add it. For example, in .NET you might do this:
 
 ```csharp
 var accountId = Baggage.GetBaggage("AccountId");
 Activity.Current?.SetTag("AccountId", accountId);
 ```
 
+Because one of the most common use cases for Baggage is to add data to
+[Span Attributes](/docs/concepts/signals/traces/#attributes) across a whole
+trace, several languages have Baggage Span Processors that automatically read
+Baggage values and add them as attributes for you.
+
+## Baggage is only for observability data
```

Comment:
any reason why we need to mention that? 

For example, if a user is using baggage to pass something like ("real-user","false") to all downstreams, and downstream services could be leveraging this to make decisions like which traffic to prioritize (eg: prioritize ("real-user","true")), without ever recoding the baggage in any telemetry data (i.e Baggage is not ending up on Observability data), is that wrong?

OTel Baggage spec (or W3CBaggage) does not have any such wordings, so I don't think we should add it for the docs here.

Reply:
Yeah, it's not mentioned in the spec but I really think we don't want people using this as a replacement for a data flow library or framework. Storing potentially large amounts of data in HTTP headers via a mechanism designed for observability feels wrong.

Reply:
we should mention it because we don't want people to do it. i've seen people abuse baggage then get confused by it. if someone wants to use w3c baggage for _other_ reasons, fine, but they should do it outside of otel namespaces

Reply:
if doc owners agree with this, I have no objection to add it here. But I'd suggest to first add such wording in the spec itself, instead of just adding in the docs.

One interesting wording I noticed [in the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-a-tracer):
_"OpenTelemetry API may also ignore this name and return a default instance for all calls, if it does not support "named" functionality **(e.g. an implementation which is not even observability-related**)"_

Reply:
@kalyanaj To share any comments from W3CBaggage perspective, which could be different from OTel's perspective.

Reply:
From the W3C baggage spec perspective, we think of two categories of use cases for distributed context propagation / baggage: 

1) use cases that improve observability and 
2) use cases that improve controllability. 

We have documented some example use cases here: https://github.com/w3c/baggage/blob/main/baggage/README.md, and this is inspired from @yurishkuro 's post on the same topic.  Also, ccing: @dyladan and @SergeyKanzhelev. 

So, if someone wants to accomplish the use cases in category 2 above, my understanding until now had been that OTel Baggage APIs could be used for it: if the customer rolls out their own libraries in multiple languages instead of using the standard OTel libraries, I feel that will lead to even more problems. Are the possibilities of misuse specific to the non-observability use cases, or can they happen in either category?

So, want to learn/understand if there are other mechanisms to prevent abuse - e.g., by having suitable limits (which I believe OTel already has), semantic conventions for well-known use cases etc.

Reply:
I know many use cases from real large scale systems that use baggage for things outside of observability.  Just recently saw a huge overload protection win in a team that used baggage to propagate request priority.

Reply:
Okay. Sounds good to me. I think we can just nix this section.